xm-commons-lep/src/main/java/com/icthh/xm/commons/lep/api/BaseLepContext.java

```diff
@@ -16,6 +18,7 @@ public abstract class BaseLepContext {
     public BaseProceedingLep lep;
     public LepThreadHelper thread;
     public XmAuthenticationContext authContext;
+    public XmAuthenticationContextHolder authContextHolder;
```

Comment:
is it okay that this value is not explicitly initialized anywhere? 

Shouldn't it be initialized in LepContextServiceImpl#createLepContext like this? 

```java
baseLepContext.authContextHolder = xmAuthContextHolder;
```

Reply:
fixed